### PR TITLE
Address dependabot alerts

### PR DESCRIPTION
**Pull request by tnorling**
Addresses dependabot alerts

### CODE REVIEW

File: lib/msal-browser/package-lock.json [hunk 1]
```diff
@@ -4971,7755 +5552,3464 @@
...
-            "slash": "^3.0.0"
-          }
-        },
-        "jest-resolve-dependencies": {
-          "version": "27.0.4",
-          "dev": true,
-          "requires": {
-            "@jest/types": "^27.0.2",
-            "jest-regex-util": "^27.0.1",
-
```
Comment (peterzenz) [inline lib/msal-browser/package-lock.json:12703]: Is this a downgrade v4.0.0 to 3.1.1, or are these different modules with very similar names?

**Review by peterzenz — APPROVED** [review]

File: lib/msal-browser/package-lock.json (hunk 1 above)
Comment (peterzenz) [reply lib/msal-browser/package-lock.json:12703]: The context above is poor.  Here's the diff:
```
--    "yallist": {
--      "version": "4.0.0",
++    "node_modules/yallist": {
++      "version": "3.1.1",
++      "resolved": "https://registry.npmjs.org/yallist/-/yallist-3.1.1.tgz",
++      "integrity"

Comment (tnorling) [reply lib/msal-browser/package-lock.json:12703]: Appears both versions are used by different dependencies. Package-lock diffs can be hard to read as things tend to move around

```
│       └── yallist@4.0.0
├─┬ @babel/core@7.21.4
│ └─┬ @babel/helper-compilation-targets@7.21.4
│   └─┬ lru-cache@5.1.1
│     └── yallist@3.1.1
├─┬ jest@27.5.1
│ └─┬ @jest/core@27.5.1
│   └─┬ jest-snapshot@27.5.1
│     └─┬ semver@7.4.0
│       └─┬ lru-cache@6.0.0
│         └── yallist@4.0.0
├─┬ ssri@8.0.1
│ └─┬ minipass@3.3.6
│   └── yallist@4.0.0
└─┬ ts-jest@27.1.5
  └─┬ semver@7.4.0
    └─┬ lru-cache@6.0.0
      └── yallist@4.0.0
```

Comment (peterzenz) [reply lib/msal-browser/package-lock.json:12703]: Got it - thanks!

**Review by derisen — APPROVED** [review]

**Pull request merged**